tests/test_routes.py

```diff
@@ -0,0 +1,153 @@
+
+    # Assert
+    assert response.status_code == 200
+    assert response_body == []
+
+def test_read_one_planet(client, two_saved_planets):
+
+    response = client.get("/planets/2")
+    response_body = response.get_json()
+
+    assert response.status_code == 200
+    assert response_body == {
+        "id": 2,
+        "name": "Earth",
+        "description": "a little polluted",
+        "color": "purple"
+    }
+def test_create_planet(client):
+    # Arrange
+    new_planet = {
+        "name": "Colorful Planet",
+        "description": "The Best!",
+        "color": "rainbow"
+    }
+    # Act
+    response = client.post("/planets", json=new_planet)
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 201
+    assert response_body == "Planet Colorful Planet successfully created"
+
+def test_update_planet(client, two_saved_planets):
+    # arrange
+    updated_planet = {
+        "name": "Zaelas Home",
+        "color": "Cinnamon spice and everything nice",
+        "description":"a baby planet"
+    }
+    #  act
+    response = client.put("/planets/1", json=updated_planet)
+    response_body = response.get_json()
+
+    # assert
+    assert response_body == "planet #1 successfully updated"
+    assert response.status_code == 200
+
+def test_update_planet_invalid_id(client, two_saved_planets):
+    # arrange
+    updated_planet = {
+        "name": "Zaelas Home",
+        "color": "Cinnamon spice and everything nice",
+        "description":"a baby planet"
+    }
+    #  act
+    response = client.put("/planets/lala", json=updated_planet)
+    response_body = response.get_json()
+
+    # assert
+    assert response_body == {"message": "Planet lala invalid"}
+    assert response.status_code == 400
+
+def test_update_planet_extra_keys(client, two_saved_planets):
+    # arrange
+    updated_planet = {
+        "name": "Zaelas Home",
+        "color": "Cinnamon spice and everything nice",
+        "description":"a baby planet"
+    }
+    #  act
+    response = client.put("/planets/1", json=updated_planet)
+    response_body = response.get_json()
+
+    # assert
+    assert response_body == "planet #1 successfully updated"
+    assert response.status_code == 200
+    
+def test_update_planet_not_in_database(client, two_saved_planets):
+    # arrange
+    updated_planet = {
+        "name": "Zaelas Home",
+        "color": "Cinnamon spice and everything nice",
+        "description":"a baby planet"
+    }
+    #  act
+    response = client.put("/planets/3", json=updated_planet)
+    response_body = response.get_json()
+
+    # assert
+    assert response_body == {"message":"Planet 3 not found"}
+    assert response.status_code == 404
+    
+def test_delete_planet(client, two_saved_planets):
+    response = client.delete("/planets/1")
+    response_body = response.get_json()
+
+    assert response_body == "planet #1 successfully deleted"
+    assert response.status_code == 200
```

Comment:
It would be a good idea to follow this up with a get call to confirm that the planet actually got removed from the database.